compiler/src/dotty/tools/dotc/core/ConstraintHandling.scala

```diff
@@ -647,9 +647,9 @@ trait ConstraintHandling {
    * At this point we also drop the @Repeated annotation to avoid inferring type arguments with it,
    * as those could leak the annotation to users (see run/inferred-repeated-result).
    */
-  def widenInferred(inst: Type, bound: Type, widenUnions: Boolean)(using Context): Type =
+  def widenInferred(inst: Type, bound: Type, widen: Widen)(using Context): Type =
```

Comment:
The docs have not been updated with the renaming (see L633 and L636).

Reply:
I would also expect the text describing the semantics of the widening. Reading ahead there seems to be some new interaction between `widen` and `bound` when the latter is a singleton.